app/main.py

```diff
@@ -1,6 +1,22 @@
+import os
+import requests
+
+
+API_KEY = os.environ.get("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json?"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    result = requests.get(URL + f"key={API_KEY}&q={FILTERING}").json()
+    city = result["location"]["name"]
```

Comment:
what if API returns 500 error? Or 400 or 401? You won't get data and error will be raised